src/main/java/org/akaza/openclinica/domain/enumsupport/StudyEventWorkflowEnum.java

```diff
@@ -0,0 +1,31 @@
+package org.akaza.openclinica.domain.enumsupport;
+
+import core.org.akaza.openclinica.i18n.util.ResourceBundleProvider;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ResourceBundle;
+
+/**
+ * The StudyEventWorkflowEnum enumeration.
+ */
+public enum StudyEventWorkflowEnum {
```

Comment:
can you rename this to StudyEventWorkflowStatusEnum since this is representing the status?

Reply:
done